content/docs/command-reference/repro.md

```diff
@@ -10,14 +10,17 @@ analyzing dependencies and <abbr>outputs</abbr> of the target stages.
 ```usage
 usage: dvc repro [-h] [-q | -v] [-f] [-s] [-c <path>] [-m] [--dry] [-i]
                  [-p] [-P] [-R] [--no-run-cache] [--force-downstream]
-                 [--no-commit] [--downstream] [--pull]
-                 [targets [targets ...]]
+                 [--no-commit] [--downstream] [--pull] [--glob]
+                 [targets [<target> ...]]
 
 positional arguments:
-  targets        Stage or path to dvc.yaml or .dvc file to reproduce. Using -R,
-                 directories to search for stages can also be given.
+  targets        Stage or path to dvc.yaml or .dvc file to reproduce.
+                 If no targets are provided, it is assumed to be the
+                 dvc.yaml present in the current working directory.
```

Comment:
I don't see this change in https://github.com/iterative/dvc/pull/4976/files. Seems unrelated to the PR?

Reply:
Sorry about this. This one was missing and as I already had a big change, so I thought why not I add a one-liner for this.

Reply:
Fine by me as long as it's actually part of the help output now 🙂 

It's a strange change though, since `targets` is an optional arg I'm not sure its necessary to say what the default behavior is there, and probably inconsistent with other commands that also use `targets`.

Reply:
p.s. currently on DVC 1.10.1 I get `Stages to reproduce. 'dvc.yaml' by default.`

Reply:
Same for the master branch.

Reply:
Could we go for this?
```suggestion
  targets        Stage name or path to dvc.yaml or .dvc file to reproduce.
                 ./dvc.yaml by default
```

Reply:
Continuing discussion in https://github.com/iterative/dvc.org/pull/1983#pullrequestreview-550965610